controllers/profiles/removeImageProfile.js

```diff
@@ -1,11 +1,11 @@
 const {User} = require('../../databases/models');
-const Validator = require('fastest-validator');
 const moment = require('moment');
-const v = new Validator();
 const updateUser = require('../../services/getstream/updateUser');
+const {USERS_DEFAULT_IMAGE} = require('../../helpers/constants');
 
 module.exports = async (req, res) => {
   try {
+    let defaultImage = USERS_DEFAULT_IMAGE;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `defaultImage` variable is declared but not used within the scope of this code hunk. Ensure that it's being utilized correctly in the subsequent code or consider removing it if it's unnecessary.
```diff
- let defaultImage = USERS_DEFAULT_IMAGE;
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->